Samples/WindowsCopilotRuntime/cs-winui/Pages/LanguageModelPage.xaml

```diff
@@ -35,13 +35,13 @@
             <TextBlock Text="Language Model"
                     Style="{StaticResource TitleTextBlockStyle}"
                     Margin="10"/>
-            <TextBlock Text="The Language Model APIs let you complete text and generate embedding vectors from a given string of text."
+            <TextBlock Text="The Language Model APIs let you complete text and generate embedding vectors from a given string of text. Currently, only Text Intelligence skills are available in the stable version. Switch to feature/experimental branch to test out the experimental features of Windows AI Apis."
                     Style="{StaticResource BodyTextBlockStyle}"
                     Margin="10"/>
 
             <controls:ModelInitializationControl SourceFile="Examples/LanguageModelInit.md"/>
-
-            <TextBlock Text="Text Completion"
+            <!---  Language Model apis currently are not available in stable version of winappsdk. Switch to experimental version to enable them.  -->
```

Comment:
if this code is enabled in release/experimental, a brief comment is fine and all the disabled Xaml should be removed.